libssu/ssurepomanager.cpp

```diff
@@ -470,16 +475,14 @@ QString SsuRepoManager::url(const QString &repoName, bool rndRepo,
     }
 
     // search for URLs for repositories. Lookup order is:
-    // 1. overrides in ssu.ini
-    // 2. URLs from features
-    // 3. URLs from repos.ini
+    // 1. URLs from features
+    // 2. URLs from repos.ini
+    // 3. User URLs in ssu.ini (no override)
 
     SsuFeatureManager featureManager;
     QString r;
 
-    if (settings->contains("repository-urls/" + adaptationRepoName)) {
-        r = settings->value("repository-urls/" + adaptationRepoName).toString();
-    } else if (!featureManager.url(adaptationRepoName, rndRepo).isEmpty()) {
+    if (!featureManager.url(adaptationRepoName, rndRepo).isEmpty()) {
```

Comment:
A line before the if-statement.

```
QString repo = featureManager.url(adaptationRepoName, rndRepo);
if (!repo.isEmpty()) {
    r = repo;
}
```

Reply:
I understand what you're saying. Up to you how do you want to do it. Surely, it's not a deal breaker if we leave it like it is now.